src/System.Net.Security/ref/TlsCipherSuite.cs

```diff
@@ -0,0 +1,359 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I don't think we should be generating the ref with a T4 template.  Deciding to update public surface area should be a very manual choice.